0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -521,54 +576,46 @@ The `Message` class is used to describe local ledger message. All fields are req
 ###### Fields
 | Type | Name | Description |
 |:--|:--|:--|
-| `String` | account | ILP Address of the source or destination account (deprecated) |
-| `String` | from | ILP Address of the source account |
-| `String` | to | ILP Address of the destination account |
-| `String` | ledger | ILP Address prefix of the ledger |
-| `Object` | data | Data packet to be sent with the message |
-
-### IncomingMessage
-<code>class IncomingMessage extends [Message](#class-message)</code>
-
-`IncomingMessage` objects describe messages which are **received** by the ledger plugin. The `account` field refers to the local source account that the message originated from.
-
-See [`Message`](#class-message) for more information.
-
-### OutgoingMessage
-<code>class OutgoingMessage extends [Message](#class-message)</code>
-
-`OutgoingMessage` objects describe messages which have been **sent** by the ledger plugin. The `account` field refers to the local destination account on the underlying ledger.
-
-See [`Message`](#class-message) for more information.
-
-#### account
-<code>**account**:String</code>
-
-The ILP Address of a local account.
+| `String` | [id](#message-id) | Unique message identifier |
+| `String` | [from](#message-from) | ILP Address of the source account |
+| `String` | [to](#message-to) | ILP Address of the destination account |
+| `String` | [ledger](#message-ledger) | ILP Address prefix of the ledger |
+| `String` | [ilp](#message-ilp) | Base64-encoded ILP packet |
```

Comment:
This should clarify that "ILP Packet" in this context does not necessarily mean ILP payment packet. I was confused looking at this at first